### ISSUE TITLE
[Old Link to NVIDIA Driver] Ubuntu 16.04 (CUDA 10.1)

### ISSUE THREAD

**Issue by Rowing0914** [tensorflow/tensorflow#45221]
## URL(s) with the issue
- https://www.tensorflow.org/install/gpu

## Description of the issue:
In the section `Ubuntu 16.04 (CUDA 10.1)`, the following command is used to download the driver

```bash
wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb
```

But the link specified is actually outdated and invalid so that it has to be replaced as follows
- Old: http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb
- New: https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64/nvidia-machine-learning-repo-ubuntu1604_1.0.0-1_amd64.deb

**Comment by vishalsingha** [comment]
Hii, I would like to update this.

**Comment by ravikyram** [comment]
@Rowing0914 

This issue got merged in the [PR#1755](https://github.com/tensorflow/docs/pull/1755). Please, verify once and close the issue. Thanks!

**Comment by Rowing0914** [comment]
Thank you!!
